spec/acceptance/generate_markdown_spec.rb

```diff
@@ -31,19 +33,15 @@ class { "test": }
 
   EOF
 
-  it 'should render Markdown to stdout when using --format markdown and --stdout' do
-    test_module_path = get_test_module_path(master, /Module test/)
-    on master, puppet('strings', 'generate', '--format markdown', "#{test_module_path}/manifests/init.pp") do
-      output = stdout.chomp
-      expect(JSON.parse(output)).to eq(expected)
-    end
+  it 'should render Markdown to stdout when using --format markdown' do
+    skip('This test is broken. Does not output to STDOUT by default.')
```

Comment:
Yep, looking at the code I see why. I'll file a ticket if there isn't one already.